tartufo/commands/scan_local_repo.py

```diff
@@ -51,4 +52,4 @@ def main(
         util.fail(f"There was an error fetching from the remote repository: {exc}", ctx)
     except types.TartufoException as exc:
         util.fail(str(exc), ctx)
-    return (str(repo_path), issues)
+    return (str(repo_path), issues, scanner)
```

Comment:
I'm hesitant to change the return signature here. I see why you've done it, but my concern is that this change has only been applied to one of the commands. What about the other commands? Don't they need similar changes made?

Changing this signature just has potentially wider-reaching consequences than just this. I want to be sure we're fully aware of them before taking that step.

Reply:
I think there's value in not creating the GitRepoScanner object twice though.

Reply:
Right, to @jgowdy 's point here - the scanner contains the compiled list of all the exclusions, gathered from a traversing the different config options to look for exclusions. I see value in not replicating that logic and instead passing the scanner to the output writer